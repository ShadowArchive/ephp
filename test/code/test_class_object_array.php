<?php

class A {
    public $a;
}

$data = [ new A, new A, new A ];
for ($i=0; $i<count($data); $i++)
    $data[$i]->a = 10;

var_dump($data);

