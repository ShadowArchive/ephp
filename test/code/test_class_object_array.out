array(3) {
  [0]=>
  object(A)#1 (1) {
    ["a"]=>
    int(10)
  }
  [1]=>
  object(A)#2 (1) {
    ["a"]=>
    int(10)
  }
  [2]=>
  object(A)#3 (1) {
    ["a"]=>
    int(10)
  }
}
